core/objects.go

```diff
@@ -577,6 +577,18 @@ type FQDNSet struct {
 	Expires time.Time
 }
 
+// OrderFQDNSet contains the SHA256 hash of the lowercased, comma joined names
```

Comment:
Good call. I think I originally put it here because I thought it needed to be exported in order to add the gorp table mapping, but that happens in the SA too!

Reply:
Resolved in 1621fb8